weblate/addons/base.py

```diff
@@ -29,6 +29,8 @@
 if TYPE_CHECKING:
     from django_stubs_ext import StrOrPromise
 
+    from weblate.trans.models import Component, Project
```

Comment:
I am not sure why `ruff` moves the import inside `if`. What might be the reasoning here?

Reply:
To avoid importing it at runtime when it is needed only for type checking.